lib/templates.js

```diff
@@ -120,6 +135,33 @@ const createGroupsTemplateForService = (serviceName, serviceDefinition, appConfi
     };
 }
 
+/**
+ * This is a template function to create EntityType object for EntityTypes Array.
+ * @param { object } appConfig  The configuration object.
+ * @param { Array } packageIds The identifiers of packages.
+ * @param { object } entity
+ * @returns { object } An object for the EntityType.
+ */
+const createEntityTypeTemplate = (appConfig, packageIds, entity) => {
+    // we collect only entities annotated by @EntityRelationship.entityType,e.g. 'sap.sm:AribaEntity1'
+    return {
+        ordId: entity.ordId,
+        localId: entity.entityName,
+        title: entity["@title"] ?? entity["@Common.Label"] ?? entity.entityName,
+        shortDescription: SHORT_DESCRIPTION_PREFIX + entity.entityName,
+        description: DESCRIPTION_PREFIX + entity.entityName,
+        version: "1.0.0",
+        lastUpdate: appConfig.lastUpdate,
+        visibility: RESOURCE_VISIBILITY.public,
+        partOfPackage: _getPackageID(appConfig.ordNamespace, packageIds, ORD_RESOURCE_TYPE.entityType),
+        releaseStatus: "active",
+        level: entity["@ObjectModel.compositionRoot"] || entity["@ODM.root"] ? "root-entity" : "sub-entity", // TODO
+        extensible: {
+            supported: "no",
+        }
+    };
+};
```

Comment:
missing the ...ordExtensions

Reply:
I don't understand what would be the implication of "including" ordExtensions in case of `entityTypes`. Aren't they added on service(and not on entity) level?
Example: 
```
annotate AdminService with @ORD.Extensions: {
    title         : 'This is Admin Service title',
    industry      : [
        'Retail',
        'Consumer Products'
    ],
    lineOfBusiness: ['Sales']
};

```

Reply:
Added ...ordExtensions.

Reply:
I was thinking about the custom.ord.json extension, I thought they are there as well. We need to still allow custom.ord.json, but you are right we don't have service level ord extensions.